resources/assets/lib/user-list.tsx

```diff
@@ -44,6 +44,10 @@ function usernameSortAscending(x: User, y: User) {
   return x.username.localeCompare(y.username);
 }
 
+function rankSortDescending(x: User, y: User) {
+ return x.user_gamemode_rank > y.user_gamemode_rank; 
```

Comment:
properties added to the type should be added to `globals.d.ts`

Reply:
`compareFunction` for `Array.sort` should return a `number`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort